src/azure-cli/azure/cli/command_modules/acs/tests/latest/recordings/test_aks_create_spot_node_pool.yaml

```diff
@@ -13,8 +13,8 @@ interactions:
       ParameterSetName:
```

Comment:
Your change in test_aks_commands.py is for nodepool snapshot, so you need to update test_aks_nodepool_snapshot.yaml instead of test_aks_create_spot_node_pool.yaml

Reply:
fixed